app/frontend/layouts/themes/custom_styles.css

```diff
@@ -327,3 +327,8 @@
     background-repeat: no-repeat;
   }
 }
+
+.embed-responsive-item {
+  width: "100%";
+  height: clamp(200px, 55vw, 489px);
+}
```

Comment:
@pupilfirst/designers, can you review this?

Reply:
Remove these styles. They are overriding the styles defined in `markdownIt.js` 

``` diff
- .embed-responsive-item {
-   width: "100%";
-   height: clamp(200px, 55vw, 489px);
- }
```